SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.cpp

```diff
@@ -141,75 +147,61 @@ void ASpatialTestLoadBalancingData::PrepareTest()
 		FinishStep();
 	});
 
-	constexpr float ActorReceiptTimeout = 5.0f;
-	AddStep(
-		TEXT("Retrieve actors on all workers"), FWorkerDefinition::AllWorkers, nullptr, nullptr,
-		[this](float) {
-			TargetActor = GetWorldActor<ASpatialTestLoadBalancingDataActor>(GetWorld());
-			RequireTrue(TargetActor.IsValid(), TEXT("Received main actor"));
+	AddStep(TEXT("Retrieve actors on all workers"), FWorkerDefinition::AllWorkers, nullptr, nullptr, [this](float) {
+		TargetActor = GetWorldActor<ASpatialTestLoadBalancingDataActor>(GetWorld());
```

Comment:
I am not sure how this fixes the flaky test? It will still fail on RequireTrue if you get nullptr back bc its not ready?

Reply:
ah nvm I got it, we try until the Actor is ready through RequireTrue